package.json

```diff
@@ -70,13 +70,13 @@
   },
   "license": "MIT",
   "dependencies": {
-    "jquery": ">=1.9.1",
+    "jquery": ">=3.2.1",
```

Comment:
This should stay as it is. Bootstrap v4 works with versions 1.9.1 of jQuery and up